lib/ClangImporter/ImportDecl.cpp

```diff
@@ -4761,7 +4762,7 @@ namespace {
                                    Optional<AccessorInfo> accessorInfo) {
       // We can't safely re-use the same declaration if it disagrees
       // in accessor-ness.
-      auto accessor = dyn_cast<AccessorDecl>(fn);
+      auto accessor = dyn_cast_or_null<AccessorDecl>(fn);
```

Comment:
I had a hard time understanding the new semantics of `isAccceptableResult` with this patch. It looks like it will return `true` when fn is null, which is what we want, but it's not very clear from the code. What if we rename this function to `isAcceptableResultOrNull` and adding an early return for `fn == nullptr` instead of using `dyn_cast_or_null`?

Reply:
Cool yeah. That'd be less of a subtile meaning here.